src/codegen_call_json.cpp

```
@@ -10,201 +19,204 @@ static bool lookupJsonSet(const std::unordered_set<llvm::Value*> &set, llvm::Val
+        return cg.builder_.CreateCall(fn, {val}, "json_stringify");
     }
     if (e.args.size() == 2) {
-        llvm::Value *val = emitExpr(*e.args[0]);
-        if (!isJsonValue(val))
-            codegenError("stringify() requires a JsonValue as first argument");
-        llvm::Value *indent = emitExpr(*e.args[1]);
-        auto fnTy = fnTy_ptr_i64_to_ptr_;
-        auto fn = mod_->getOrInsertFunction("__ry_json_stringify_pretty", fnTy);
-        return builder_.CreateCall(fn, {val, indent}, "json_stringify_pretty");
+        llvm::Value *val = cg.emitExpr(*e.args[0]);
+        if (!cg.isJsonValue(val))
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)